packages/client.api/src/actions/ActionResults.ts

```diff
@@ -40,16 +40,12 @@ interface LargeScaleObjectEdits {
   deletedLinksCount?: never;
 }
 
-type ObjectEdit =
-  | ({ type: "addObject" } & ObjectReference)
-  | ({ type: "modifyObject" } & ObjectReference)
-  | ({
-    type: "addLink";
-    linkTypeApiNameAtoB: string;
-    linkTypeApiNameBtoA: string;
-    aSideObject: ObjectReference;
-    bSideObject: ObjectReference;
-  });
+type LinkReference = {
```

Comment:
nit for followup, types that can be interfaces are generally better as such. Its a bit easier on the typescript compiler